src/models/apigateway-base.ts

```diff
@@ -2,10 +2,20 @@ import DomainInfo = require("./domain-info");
 import ApiGatewayMap = require("./api-gateway-map");
 import DomainConfig = require("./domain-config");
 import {Client} from "@aws-sdk/smithy-client";
+import {ConfiguredRetryStrategy} from "@aws-sdk/util-retry";
 
 abstract class APIGatewayBase {
     public apiGateway: Client<any, any, any, any>;
 
+    public static getRetryStrategy() {
+        return new ConfiguredRetryStrategy(
+            5, // max attempts.
+            // This example sets the backoff at 100ms plus 5s per attempt.
+            // https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/modules/_aws_sdk_util_retry.html#aws-sdkutil-retry
+            (attempt: number) => 100 + attempt * 5000 // backoff function.
```

Comment:
Maybe should increase the backoff. We could lower the base amount to make up for it